modules/runner/src/testFixtures/java/org/apache/ignite/internal/testframework/TestIgnitionManager.java

```diff
@@ -66,7 +66,8 @@ public class TestIgnitionManager {
             "ignite.storage.profiles.default_aimem.initSize", Integer.toString(256 * MiB),
             "ignite.storage.profiles.default_aimem.maxSize", Integer.toString(256 * MiB),
             "ignite.storage.profiles.default.engine", "aipersist",
-            "ignite.storage.profiles.default.size", Integer.toString(256 * MiB)
+            "ignite.storage.profiles.default.size", Integer.toString(256 * MiB),
+            "ignite.system.properties.aipersistThrottling", "disabled"
```

Comment:
Do we have any integration test where throttling is enabled?

Reply:
No, I don't think so, we rely on "unit" tests